src/NUnitFramework/framework/Internal/TestListener.cs

```diff
@@ -21,46 +21,28 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
-using System;
 using NUnit.Framework.Interfaces;
 
 namespace NUnit.Framework.Internal
 {
     /// <summary>
-    /// TestListener provides an implementation of ITestListener that
-    /// does nothing. It is used only through its NULL property.
+    /// Provides an implementation of <see cref="ITestListener"/> through
+    /// its <see cref="NULL"/> property that does nothing.
     /// </summary>
-    public class TestListener : ITestListener
+    public static class TestListener
```

Comment:
The side effect here is that .ToString() gives `NullTestListener` and you check `== TestListener.NULL` rather than `is TestListener`. I was motivated to do this since `is TestListener` doesn't read as though you are checking for a *null* test listener.